e2e/Makefile

```diff
@@ -192,14 +219,28 @@ daemonset-lvmd/delete-minikube:
 # Set scheduler configs manually because minikube can't edit scheduler configs.
 .PHONY: daemonset-lvmd/update-minikube-setting
 daemonset-lvmd/update-minikube-setting: daemonset-lvmd/delete-minikube daemonset-lvmd/launch-minikube
+ifeq ($(STORAGE_CAPACITY),false)
```

Comment:
It would be better to use `ifneq` or comparing to `true`, IMO.

Reply:
improved it.
https://github.com/topolvm/topolvm/pull/315/commits/eb48cda127de7da48b550aa58f7236ab03777a6d